charts/sonarqube-dce/values.yaml

```diff
@@ -440,6 +440,15 @@ service:
   # loadBalancerSourceRanges:
   #   - 0.0.0.0/0
   # loadBalancerIP: 1.2.3.4
+
+# Those proxy settings will be propagated to the install-plugin and prometheus-exporter init containers
+# if httpProxySecret is set the other one will be ignored
+# the secret should contain exactly the keys http_proxy, https_proxy and no_proxy
+httpProxySecret: ""
```

Comment:
hmm, that's the kind of thing we should have on the documentation maybe? because so far, for every other secret, we have never given examples.

We could probably put this topic in the pain-point, or at least improving our tutorials.